### ISSUE TITLE
More detailed error message when login fails

### ISSUE THREAD

**Issue by jrabensc** [wallabag/ios-app#347]
| Q             | A
| ------------- | ---
| Device?       | iPhone
| Device Os version    | 14.6
| App Version   | 6.0.4 build 304
| Wallabag server version | 2.4.2

### Steps to reproduce/test case

Add new Wallabag instance, insert client id and secret and try to log in using login name and password. When something fails, the app just displays a generic error. It's not clear what exactly is wrong (client id, secret, password, name).

### Improvement
More detailed error message.

**Comment by nbgibson** [comment]
I've migrated my WB docker instance to a new host and am encountering the generic error issue again. If you need a tester feel free to reach out.

**Comment by blacs30** [comment]
Same here. I am unable to login from iOS version 6.0.4 to Wallabag Server 2.4.2 and don't know why. It works fine to Wallabag Server 2.4.1.

**Comment by bourvill** [comment]
> Same here. I am unable to login from iOS version 6.0.4 to Wallabag Server 2.4.2 and don't know why. It works fine to Wallabag Server 2.4.1.

@blacs30 can you check your log on your server ?

**Comment by blacs30** [comment]
Yes, I've done that but don't really get what they try to tell me

```
2021/07/22 07:42:48 [error] 10#10: *20351 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/wallabag/var/cache/prod/doctrine/orm/Proxies/__CG__WallabagUserBundleEntityUser.php): failed to open stream: No such file or directory in /var/www/wallabag/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 204PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/wallabag/var/cache/prod/doctrine/orm/Proxies/__CG__WallabagUserBundleEntityUser.php' (include_path='.:') in /var/www/wallabag/vendor/doctrine/common/lib/Doctrine/Common/Proxy/AbstractProxyFactory.php on line 204" while reading response header from upstream, client: 10.233.90.100, server: _, request: "POST /oauth/v2/token HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "wallabag.example.com"
```

I run it on Kubernetes. I changed the image tag from 2.4.2 to 2.4.1 and all works fine from iOS perspective.

**Comment by bourvill** [comment]
you have on error on your server. 
some file are missing. Have you run the migration tool? or clean cache after install

**Comment by jwiegley** [comment]
I just installed the Docker image `wallabag/wallabag:latest`, and haven't done anything to the image.
